gamestream_launchpad.py

```diff
@@ -16,12 +16,18 @@ def windowEnumerationHandler(hwnd, top_windows):
     top_windows.append((hwnd, win32gui.GetWindowText(hwnd)))
 
 
-def set_resolution(gamestream_width, gamestream_height):
-    print("Switching resolution to {0}x{1}".format(gamestream_width, gamestream_height))
+def set_resolution(gamestream_width, gamestream_height,refresh_rate=None):
+    if refresh_rate is None:
+        print("Switching resolution to {0}x{1}".format(gamestream_width, gamestream_height))
+    else:
+        print("Switching resolution to {0}x{1} at {2}Hz".format(gamestream_width, gamestream_height))

```

Comment:
The last argument, `refresh_rate` is missing from the `format` call here

Reply:
Yeah originally I didn't have an updated print statement and then forgot to add the value when I rushed it in. This is actually already fixed it's the second commit in this pull request.